backend/operations_scanner.go

```diff
@@ -93,9 +97,11 @@ func (s *OperationsScanner) Join() {
 	<-s.done
 }
 
-func (s *OperationsScanner) pollDBOperations(ctx context.Context, logger *slog.Logger) {
+func (s *OperationsScanner) pollDBOperations(logger *slog.Logger) {
 	var activeOperations []*database.OperationDocument
 
+	ctx := context.Background()
```

Comment:
I think we can preserve the root context here, because we aren't issuing a write or other non-interruptable operation with the context (it looks like it's just a List + Iterate - safe to cancel?)

Reply:
Agree, this function is safe to insta-terminate.